README.markdown

```diff
@@ -49,10 +49,15 @@ PusherFake::Channel.reset
 
 ### Development Environment
 
-In a Rails initializer, or any file executed during loading:
+In a Rails initializer, or any file executed during loading do the following. Please note that requiring that file immediately starts up the websocket server.
 
 ```ruby
-require "pusher-fake/support/base"
+if Rails.env == "development"
```

Comment:
Good call on providing an environment check. You could use the inquirer method:

```ruby
if Rails.env.development?
  require "pusher-fake/support/base"
end
```

And I definitely like the idea of providing defaults for the `Pusher` configuration. I'm up for setting them in the base support file when not present.

I'm also up for making these suggested changes myself if you'd like.